google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/CloudSdkAppEngineHelper.java

```diff
@@ -305,20 +322,13 @@ public void deleteCredentials() {
   @NotNull
   private DeploymentOperationCallback wrapCallbackForUsageTracking(
       final DeploymentOperationCallback deploymentCallback,
-      AppEngineDeploymentConfiguration deploymentConfiguration,
       AppEngineEnvironment environment) {
 
     StringBuilder labelBuilder = new StringBuilder();
     if (environment == AppEngineEnvironment.APP_ENGINE_STANDARD) {
       labelBuilder.append("standard");
     } else {
       labelBuilder.append("flex");
-
-      if (deploymentConfiguration.isAuto()) {
-        labelBuilder.append(".auto");
-      } else {
-        labelBuilder.append(".custom");
```

Comment:
We got rid of auto and custom configuration for deployments. Configs are always custom from now on.